DEPLOYMENT.md

```diff
@@ -42,7 +42,7 @@ One for the Java server, one for hosting the client files, and one for the datab
 
 ## Step 2: Creating a droplet
 
-- Go to [this link](https://cloud.digitalocean.com/droplets/new?image=docker-18-04&app=docker&size=s-1vcpu-1gb). It should bring you to the Create Droplets page with the Docker marketplace image selected and the $5/month standard plan selected. If those are not selected, please select them.
+- Go to [this link](https://cloud.digitalocean.com/droplets/new?image=docker-18-04&app=docker&size=s-1vcpu-1gb&options=install_agent). It should bring you to the Create Droplets page with the Docker marketplace image selected and the $5/month standard plan selected. If those are not selected, please select them.
```

Comment:
Probably want to change `docker-18-04` to `docker-20-04` (or whatever is current and appropriate).